src/extension/inpage-script/index.js

```diff
@@ -45,10 +45,12 @@ function registerLightningLinkClickHandler() {
         return;
       }
       const lightningLink = target.closest('[href^="lightning:" i]');
-      const lnurlLink = target.closest('[href^="lnurl" i]');
+      const lnurlLink = target.closest(
```

Comment:
Could we link to the relevant spec? I think it is LUD-17

Reply:
Should we support keyauth too?

Reply:
> Could we link to the relevant spec? I think it is LUD-17

Done in 385af36

> Should we support keyauth too? (I think this change will currently break it)

I think you never supported keyauth so this change will not break it. I searched for `keyauth` in your source code and wasn't found:

```
$ grep -R "keyauth" src/
```

Reply:
Ah ok, thanks! I'll made a new issue for that: https://github.com/getAlby/lightning-browser-extension/issues/2984